src/components/scenes/FioConnectWalletScene.js

```diff
@@ -129,7 +133,7 @@ export class FioConnectWalletScene extends Component<Props, State> {
   render () {
     const { notConnectedWallets } = this.props
     const { selectedWalletsMap, fioWalletLoading } = this.state
-    const continueDisabled = !Object.keys(selectedWalletsMap).length
+    const continueDisabled = !Object.keys(selectedWalletsMap).length || Object.keys(selectedWalletsMap).length > 5
```

Comment:
Not sure I like the limitation of only allowing 5 wallets to be selected at a time. Why did you add this?

Reply:
The FIO call to add addresses limits a user to adding 5 at time. This was done to prevent abuse of "feeless" call because it is bundle eligible. So, you can add more than 5, but only 5 at a time right now.